config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'users#index'
+  get 'users/:id', to: 'users#show'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/:id/posts/:post_id', to: 'posts#show'
 end
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍